topdown/http.go

```diff
@@ -88,7 +88,11 @@ func validateHTTPRequestOperand(term *ast.Term, pos int) (ast.Object, error) {
 // Adds custom headers to a new HTTP request.
 func addHeaders(req *http.Request, headers map[string]interface{}) {
 	for k, v := range headers {
-		req.Header.Add(k, v.(string))
+		// Type assertion
+		header, ok := v.(string)
```

Comment:
I think we should return an error here like we do below.

Reply:
okay, will fix it although I have mixed feelings about this.  If somebody passed 10 headers and just one is invalid maybe logging would be less disruptive.

Reply:
Silently discarding headers could lead to difficult to debug situations. I think we're better off returning an error so that the problem is surfaced.

Reply:
okay, done